doc/source/security/data-masking.rst

```diff
@@ -0,0 +1,307 @@
+
+Use a selected dictionary to generate random terms. The dictionary must
+be loaded from a file with the following characteristics:
+
+* Plain text
+
+* One term per line
+
+* Must contain at least one entry
+
+The following table displays the commands for using dictionaries to generate random terms:
+
+.. tabularcolumns:: |p{0.35\linewidth}|p{0.15\linewidth}|p{0.50\linewidth}|
+
+.. list-table::
+    :widths: 20 30 60
+    :header-rows: 1
+
+    * - Parameter
+      - Description
+      - Sample
+    * - gen_blacklist(str, dictionary_name, replacement_dictionary_name)
+      - Replaces a term with a term from a second dictionary.
+
+      - .. code-block:: MySQL
+
+            mysql> SELECT gen_blacklist('apple', 'fruit', 'nut');
+
+            +-----------------------------------------+
+            | gen_blacklist('apple', 'fruit', 'nut')  |
+            +-----------------------------------------+
+            | walnut                                  |
+            +-----------------------------------------+
+
+    * - gen_dictionary(dictionary_name)
+      - Returns a random term from the selected dictionary.
+
+      - .. code-block:: MySQL
+
+            mysql> SELECT gen_dictionary('trees');
+
+            +--------------------------------------------------+
+            | gen_dictionary('trees')                          |
+            +--------------------------------------------------+
+            | Norway spruce                                    |
+            +--------------------------------------------------+
+
+    * - gen_dictionary_drop(dictionary_name)
+      - Removes the selected dictionary from the dictionary registry. Returns
+        either success or failure.
+
+      - .. code-block:: MySQL
+
+          mysql> SELECT gen_dictionary_drop('mytestdict')
+
+          +-------------------------------------+
+          | gen_dictionary_drop('mytestdict')   |
+          +-------------------------------------+
+          | Dictionary removed                  |
+          +-------------------------------------+
+
+    * - gen_dictionary_load(dictionary path, dictionary name)
+      - Load a file into the dictionary registry and configures the dictionary
+        name. The name can be used with any function. If the dictionary is
+        edited, you must drop and then reload the dictionary to view the changes.
+        Returns either success or failure.
+
+      - .. code-block:: MySQL
+
+             mysql> SELECT gen_dictionary_load('/usr/local/mysql/dict-files/testdict', 'testdict');
+
+             +------------------------------------------------------------------------+
+             | gen_dictionary_load('/usr/shared/mysql/dict-files/fndict', 'fndict')   |
+             +========================================================================+
+             | Dictionary load successfully                                           |
+             +------------------------------------------------------------------------+
```

Comment:
change this entire order of this section. It should be as if the user tries to use this functionality
1. creating dictionary files
2. gen_dictionary_load()
3. gen_dictionary()
4. gen_dictionary_blacklist()
5. gen_dictionary_drop().

Reply:
Restructured.